### PR TITLE
Add pages for reopenInvestigation and taskComplete

### PR DESCRIPTION
**Pull request by tschanfeld**
Requested here https://github.com/demisto/etc/issues/45988 to be included in the builtins section of xsoar.pan.dev below Get Entries Filter topic. Includes info from script helper as well as info about basic use cases.

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/task-complete.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+id: taskComplete 
+title: taskComplete 
+---
+You can use this command to reopen a closed incident. For example, you can use the reopenInvestigation command to add an action button in the incident layout, that can reopen a closed incident and take a specific action, such as rerunning the incident's playbook. You can also include the reopenInvestigation command in a loop in a script to reopen multiple incidents. 
+
+| Argument         | Description           | 
+| ------------- |-------------|   
+| id     | Which incident to reopen. If no incident ID is provided, the current incident is reopened.      |   
+
```
Comment (ali-sawyer) [inline docs/integrations/task-complete.md:10]: Looks like the wrong description (the one for `reopenInvestigation`) got copied here.

**Review by ali-sawyer — CHANGES_REQUESTED** [review]
`reopenInvestigation` looks good. Please fix the description for `taskComplete`. Thanks!

**Review by ali-sawyer — APPROVED** [review]
Looks good!

**Comment by tschanfeld** [comment]
@ali-sawyer Can you tell what went wrong here?  ci/circleci: build — Your tests failed on CircleCI

**Comment by ali-sawyer** [comment]
There was an error with some code CircleCI runs, unrelated to the change we made. A rerun might fix it but I don't have permissions to do so. 

Screen Shot 2022-03-14 at 5 14 19 PM
Screen Shot 2022-03-14 at 5 14 38 PM

Do you know who on the Engineering side administrates CircleCI?

**Comment by tschanfeld** [comment]
@ShahafBenYakir Can you assist?

**Comment by ChanochShayner** [comment]
@MosheEichler fyi

**Pull request merged**